playbooks/roles/hxat/templates/secure.py.j2

```diff
@@ -0,0 +1,20 @@
+SECURE_SETTINGS = {
```

Comment:
I have no clue what this PR is really about but I wanted to take a poke. Is there any reason these `.py` files aren't a part of the codebase, using a `.env` file to gather their settings from, where we only render the `.env` file through Ansible?

Reply:
This secure.py file does the same role as the .env file, it concentrates all the custom settings in one place (well, almost all the settings we want to change). I [saw it used](https://github.com/harvard-vpal/bridge-adaptivity/blob/master/bridge_adaptivity/config/settings/secure_example.py) in another Harvard project.

Reply:
We tend to follow 12-factor-app, which has this to say about configuration: https://12factor.net/config. This is the pattern we (and edX) use for basically everything config-wise, and is the norm in almost all of our deployments - which is why I find rendering full Python files for the settings to be a bit awkward to do.

Anyway, I'm not involved in this project, just a suggestion.

Reply:
Yes, I also don't like it and I pointed it out in the ticket. If we need to migrate often, we may need to improve this.
Now, we'd have to rewrite their code to read environment variables instead of their secure.py file, and contribute this upstream (to the Harvard project), and we didn't include this in the discovery.
Note that the this template inside the ansible role doesn't set hardcoded values, but it accesses ansible variables that are defined outside the role (e.g. in ansible-secrets).

Reply:
Thanks for the suggestion, @UmanShahzad 
We discussed this awkward setup which is used by Harvard, and it would take more than a few changes to move away from it as mentioned by @clemente